packages/lesswrong/lib/editor/make_editable.ts

```diff
@@ -117,6 +117,8 @@ const buildEditableResolver = <N extends CollectionNameString>(
         on: (revisionField) => `CASE WHEN ${resolverArg("version")} IS NULL
           THEN
             ${field(`${fieldName}_latest` as FieldName<N>)} = ${revisionField("_id")}
+          WHEN ${resolverArg("version")} = 'draft' THEN
+            ${revisionField("_id")} = (SELECT _id FROM "Revisions" WHERE "documentId" = ${field("_id")} AND "fieldName" = '${fieldName}' ORDER BY "editedAt" DESC LIMIT 1)
```

Comment:
I'm a bit nervous about interpolating `fieldName` here, even though it's controlled by us.  Not sure if this justifies creating an additional abstraction in `SelectFragmentQuery`.  @oetherington thoughts?

Reply:
I think it's fine in this case since, as you say, it's safe input. But maybe we should add a warning comment explitily saying _why_ it's safe and that people in the future shouldn't copy/paste it thinking that it's an example of how things should be done generally.